pim-tool-back/src/main/java/vn/elca/training/web/ProjectController.java

```diff
@@ -1,32 +1,67 @@
 package vn.elca.training.web;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+
 import vn.elca.training.model.dto.ProjectDto;
+import vn.elca.training.model.entity.Project;
 import vn.elca.training.service.ProjectService;
-import vn.elca.training.util.ApplicationMapper;
-
-import java.util.List;
-import java.util.stream.Collectors;
 
 /**
- * @author gtn
- *
+ * @author thomas.dang
+ * 
  */
 @RestController
-@RequestMapping("/projects")
+@RequestMapping("/project")
```

Comment:
[N] It is OK to just keep everything as endpoint "/projects". Singular and plural does not matter much, it is mainly to grouped api by "entity".